src/components/SubmissionForm/SubmissionForm.js

```diff
@@ -25,14 +27,41 @@ const SubmissionForm = () => {
     setSongTitle(event.target.value)
   }
   const handleVideo = event => {
-    setVideo(event.target.value)
+    setVideo(URL.createObjectURL(event.target.files[0]))
```

Comment:
Still researching the best way to send the video/image files. Need to determine if it's raw or convert into a blob.